go/release-checklist.md

```diff
@@ -1,5 +1,10 @@
 ---
-# Template generated by https://github.com/coreos/repo-templates; do not edit downstream
+name: release checklist
+about: release checklist template
+title: New release for {{ git_repo }}"
```

Comment:
There's a closing quote here, but quotes shouldn't be needed unless there's a yaml reserved character "`:`"

Reply:
Yikes lol, fixed thank you